tests/test_app/test_dist.py

```diff
@@ -355,8 +450,8 @@ def test_approx_jc69_vals(_seqs1_collection):
     pdist_app = approx_pdist()
     pdists = pdist_app(jdists)
 
-    jc_app = approx_jc69()
-    jc_dists = jc_app(pdists)
+    jc_app = jaccard_dist(k=10) + approx_pdist() + approx_jc69()
+    jc_dists = jc_app(seqs)
 
     for i, j in itertools.combinations(range(len(names)), 2):
         seq1, seq2 = names[i], names[j]
```

Comment:
need to add tests that NotImplementedError is raised when seqs have wrong moltype, not raised when they have RNA or DNA moltype
```python
@pytest.mark.parametrize("moltype", ("protein", "text", "bytes")
def test_fails(moltype):
    data = [("s1", "ACGTA"),
    ("s2", "ACGTA")]
    seqs = make_unaligned_seqs(data=data, moltype=moltype)
    with pytest.raised(NotImplementedError):
        seqs.distance_matrix()
```

add a test mirroring the above that does not fail when dna or rna are the moltypes.

Reply:
Is the second test you're suggesting not already tested for considering all other tests use DNA or RNA moltype?

Reply:
no problems making it explicit though!

Reply:
Your correct for DNA! Perhaps make one of those other tests use the rna moltype?

Reply:
testing with RNA is already the case for the following:

@pytest.mark.parametrize("moltype", ("dna", "rna"))
 def test_collection_distance_matrix_same_seq(moltype):

but will add to other tests too!